src/custom-gutter-component.coffee

```diff
@@ -0,0 +1,109 @@
+{setDimensionsAndBackground} = require './gutter-component-helpers'
+
+# This class represents a gutter other than the 'line-numbers' gutter.
+# The contents of this gutter may be specified by Decorations.
+
+module.exports =
+class CustomGutterComponent
+
+  constructor: ({@gutter}) ->
+    @decorationNodesById = {}
+    @decorationItemsById = {}
+    @visible = true
+
+    @domNode = atom.views.getView(@gutter)
+    @decorationsNode = @domNode.firstChild
+
+  getDomNode: ->
```

Comment:
Or we should add it everywhere. In that case it should be named `getDOMNode` to blend in capitalization-wise.